src/pages/edge/adobe-journey-optimizer-decisioning/api-reference.md

```diff
@@ -55,7 +55,11 @@ iOS
 
 ## onPropositionsUpdate
 
-This API registers a permanent callback which is invoked whenever the Edge extension dispatches a response Event received from the Experience Edge Network upon a personalization query. The personalization query requests can be triggered by the `updatePropositions` API, Edge extension `sendEvent` API or launch consequence rules.
+This API registers a permanent callback which is invoked whenever the Edge extension dispatches a response event received from the Experience Edge Network upon a personalization query. Additionally, the callback is invoked only if the response event payload has at least one proposition with valid, non-empty offers. The personalization query requests can be triggered by the `updatePropositions` API.
```

Comment:
Saying "... invoked only if the..." sounds a little funny to me. Is it the same as if we say "... is only invoked if..."? Since that (to me) sounds a bit better, but I want to make sure that the meaning is still the same, since sometimes verbiage switches like this _do_ make a difference to the meaning.

Reply:
My intention was to convey that the callback is only called for some cases, as listed here, but not for others. Can you recommend which option would better to indicate it?

Reply:
Got it! I think then changing it to "... is only invoked if..." is fine for these instances then.